### ISSUE TITLE
Package checker's dependencies not installed with `pip install`

### ISSUE THREAD

**Issue by shangw-nvidia** [mlcommons/logging#98]
The package checker doesn't install its dependencies with

```
pip install https://github.com/mlcommons/logging/archive/refs/heads/master.zip
```

@mwawrzos

**Comment by matthew-frank** [comment]
It also depends on the following public packages, and doesn't have a requirements.txt file for pip install.
The following should be in requirements.txt:
* pyyaml
* numpy
* scipy

**Comment by shangw-nvidia** [comment]
This issue is addressed by PR #169 .

**Comment by shangw-nvidia** [comment]
Fixed in PR #169
